core/src/main/scala/sttp/tapir/RangeValue.scala

```diff
@@ -0,0 +1,26 @@
+package sttp.tapir
+
+import scala.util.Try
+
+case class RangeValue(unit: String, start: Int, end: Int) {
+  // TODO make it better
+  def validate(contentLength: Long): Boolean = if (end < contentLength) true else false
```

Comment:
how better? If it returns a boolean, the name could be better - e.g. `isValid`, so that we know what `true` means

Reply:
I don't have more ideas I leave this TODO to think about it later